plugins/base/src/main/kotlin/translators/documentables/DefaultPageCreator.kt

```diff
@@ -51,7 +51,7 @@ open class DefaultPageCreator(
             e.nameAfterClash(), contentForEnumEntry(e), setOf(e.dri), e,
             e.classlikes.renameClashingDocumentable().map(::pageForClasslike) +
                     e.filteredFunctions.renameClashingDocumentable().map(::pageForFunction) +
-                    e.properties.renameClashingDocumentable().mapNotNull(::pageForProperty)
+                    e.filteredProperties.renameClashingDocumentable().mapNotNull(::pageForProperty)
```

Comment:
It think this filters out the entries even when we have `suppressObviousMembers = true` and / or `suppressInheritedMembers = true`, shouldn't this be filtered out in `InheritedEntriesDocumentableFilterTransformer`?

Reply:
We don't have `isObviousMembers` for properties. 
It should be filtered out in `InheritedEntriesDocumentableFilterTransformer` as well if  `suppressInheritedMembers` is set (by default it's `false`). 
This filter only allows to not generate the pages for inherited properties, but the properties  are kept.

Reply:
>We don't have isObviousMembers for properties.
This is not good overall but ok, for this PR